front/src/common/SelectorSNCF.tsx

```diff
@@ -4,18 +4,26 @@ import { useTranslation } from 'react-i18next';
 import { MdArrowRight } from 'react-icons/md';
 import { getTranslationKey } from 'utils/strings';
 import cx from 'classnames';
+import { FaTrash } from 'react-icons/fa';
+import RollingStockEditorFormModal from 'modules/rollingStock/components/rollingStockEditor/RollingStockEditorFormModal';
+import { EffortCurveCondKeys } from 'modules/rollingStock/consts';
+import { useModal } from './BootstrapSNCF/ModalSNCF';
 
 const DEFAULT_SELECTORS_CLASSNAME = 'selector-SNCF';
 
```

Comment:
selectorSNCF is a common component. It shouldn't know anything about RollingStock module. Pass a more complete onItemRemoved which open the specific modal, thanks